src/Fhir.Anonymizer.Core/AnonymizationConfigurations/AnonymizerConfigurationValidator.cs

```diff
@@ -38,6 +40,38 @@ public void Validate(AnonymizerConfiguration config)
                     throw new AnonymizerConfigurationErrorsException($"{method} not support.");
                 }
             }
+
+            // Check AES key size is valid (16, 24 or 32 bytes).
+            if (!string.IsNullOrEmpty(config.ParameterConfiguration?.EncryptKey))
+            {
+                using Aes aes = Aes.Create();
```

Comment:
is there a following clause expected?

Reply:
yes. this is a simplified using statement here and dispose will happen at the end of scope of the variable aes.